TeamCode/src/main/java/org/firstinspires/ftc/teamcode/cr/CommandRunner.java

```diff
@@ -0,0 +1,29 @@
+package org.firstinspires.ftc.teamcode.cr;
+
+import static java.lang.Thread.sleep;
+
+import com.arcrobotics.ftclib.command.CommandScheduler;
+
+/**
+ * Runnable class to run FTCLib command scheduler on an \
+ * independent thread from the main opmode loop.
+ */
+public class CommandRunner implements Runnable {
+    OpModeRunningInterface runningInterface;
+
+    public CommandRunner(OpModeRunningInterface runningInterface) {
+        this.runningInterface = runningInterface;
+    }
+
+    public void run() {
+        while (!runningInterface.isOpModeRunning()) {
+            try {
+                sleep(10);// Wait for the opmode to start to start running commands
```

Comment:
Why is `sleep` used here? Would it be really unmaintainable to just run the loop?

Reply:
> Thread.sleep() releases the processor to another runnable thread or process and marks the current thread as not runnable until the sleep time expires.

Reply:
dammm, okay. Java smart like that